src/levanter/main/train_lm.py

```diff
@@ -35,10 +35,10 @@
 
 @dataclass
 class TrainLmConfig:
-    data: LMDatasetConfig = LMDatasetConfig()
-    trainer: TrainerConfig = TrainerConfig()
-    model: LmConfig = Gpt2Config()
-    optimizer: OptimizerConfig = OptimizerConfig()
+    data: LMDatasetConfig = field(default_factory=LMDatasetConfig)
+    trainer: TrainerConfig = field(default_factory=TrainerConfig)
+    model: LmConfig = field(default_factory=Gpt2Config)
+    optimizer: OptimizerConfig = field(default_factory=OptimizerConfig)
```

Comment:
These changes are because I was getting errors from using mutable default values with dataclasses. I suspect this was a change that came in with Py3.11 or something.

Reply:
yea, looks likes a 3.11 change. Until recently tensorstore didn't seem to work with 3.11 (at least on m1) so it didn't register. Thanks for the fix!